.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,9 @@
 
 ## Docs
 
-<!-- Is this a visible change? You probably need to update docs! -->
+<!-- Could this affect a user’s behavior? We probably need to update docs! -->
+<!-- If adding docs or unsure, uncomment the next line: -->
```

Comment:
```suggestion
<!-- If adding docs or unsure, uncomment the next line: -->
```

To me as a non-native speaker, the first part of the sentence ("If adding docs or unsure") is unclear. Can we rephrase that to make it crystal clear what the condition for uncommenting the following line is?

Reply:
Does this feel clearer?

```suggestion
<!-- If docs will be needed or you’re not sure, uncomment the next line: -->
```

Reply:
Yes, that's much better! Thank you :)